### DISCUSSION TITLE
Crazyradio PA not found on scan

### DISCUSSION THREAD

**Discussion by reginaldStjohn** [bitcraze/discussions#747] [Q&A] [answered]
We have had a crazyradio PA for about 6 months now. We connected to it several times on multiple computers.  Now we cannot find the radio when we hit the scan button. Is there any way to check if the firmware, address or something like that has changed?

![Crazyfilie_connct](https://github.com/bitcraze/discussions/assets/117384888/6771813b-8e7b-45d6-af3f-312f0f324fed)

**Comment by knmcguire** [comment]
Hi!  There are some conflicting things in your post:

- Is it the crazyflie that can not be found or the crazyradio you can not find through the scan? The crazyradio does not have the address, the crazyflie has.
- What feedback are you getting from the client? In the terminal of where you started the cfclient, could you see an error message that it can't find the crazyradio? Please post it here. 

Also to help you out better:
- What is the OS that you are running the cfclient on?

**Comment by reginaldStjohn** [comment]
Thank you for your response. It is the Crazyradio that will not show up in the dropdown menu after doing a scan.  Do we need to have a crazyflie powered on for the radio to appear?

We are running the client on the BitcrazyVM downloaded from the github page. Running in VMware Workstation 17

Here is the terminal output from starting and trying to scan.  There is a warning/error about main but I have run the install for the complaining dependency and it still shows up.

![image](https://github.com/bitcraze/discussions/assets/117384888/9b3f4a6e-7293-4177-a528-7cd9ebac7bf8)

**Reply by krichardsson** [reply, marked as answer]
> Do we need to have a crazyflie powered on for the radio to appear?

Yes, the dropdown is populated with the Crazyflies that was found in the scan, not the radio itself. Note that only Crazyflies with the address that is configured in the Address field will show up.
You can also connect the Crazyflie via USB and this should show up as `usb://0` (make sure to configure USB in your virtual environment as well)

> There is a warning/error about main

I think this warning only means that no Crazyflie was found on USB, and is nothing to worry about. The logging is a bit noisy and may contain messages that sounds bad, even though they are not.
